dmoj/checkers/easy.py

```diff
@@ -1,6 +1,6 @@
 def check(process_output, judge_output, **kwargs):
     import re
     from collections import Counter
-    process_all = re.sub(r'\s+', '', process_output)
-    judge_all = re.sub(r'\s+', '', judge_output)
+    process_all = re.sub(br'\s+', '', process_output)
+    judge_all = re.sub(br'\s+', '', judge_output)
```

Comment:
I have trouble believing this works. `''` is highly sketchy. [Untested too.](https://codecov.io/gh/DMOJ/judge/src/PY3/dmoj/checkers/easy.py)